modules/sql/README.md

```diff
@@ -0,0 +1,5 @@
+# @loaders.gl/sql
+
+This module contains helper classes for SQL-based loaders.
```

Comment:
Are we aware of other formats on our potential roadmap that are SQL based? If not we might just call this module `geopackage` for now?

Reply:
No I don't know of other SQL-based formats on the roadmap, but figured naming it as `sql` would have more future compatibility. Happy to rename to `geopackage`

Reply:
Yes let's do that.